docs/nitpick-exceptions

```diff
@@ -9,5 +9,11 @@ py:class pandas._libs.interval.Interval
 py:class pandas.core.frame.DataFrame
 py:class pandas.core.series.Series
 py:class pandas.core.generic.PandasObject
+
 # TODO: Need to revisit this. Nitpicking here to avoid failing builds on 3.8 and 3.9 in LBWSGRisk
-py:class PopulationView
\ No newline at end of file
+py:class PopulationView
```

Comment:
What is this file doing?

Reply:
This is making it so that the doc build doesn't fail if there are broken links pointing at something defined in this file.